### ISSUE TITLE
Angular tutorials

### ISSUE THREAD

**Issue by Reozil** [angular/angular#19877]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Feature request
[X] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
</code></pre>

## Current behavior

The tutorial https://angular.io/guide/dynamic-component-loader live demo is not working.

## Expected behavior

A working live demo

## Environment

<pre><code>
Angular version: X.Y.Z

Browser:
- [ ] Chrome (desktop) version XX
- [ ] Chrome (Android) version XX
- [ ] Chrome (iOS) version XX
- [ ] Firefox version XX
- [ ] Safari (desktop) version XX
- [ ] Safari (iOS) version XX
- [ ] IE version XX
- [ ] Edge version XX
 
For Tooling issues:
- Node version: XX  
- Platform:  

Others:

</code></pre>

**Comment by gkalpak** [comment]
Sounds like a duplicate of #19322. The example works for me. It is probably being blocked by an ad blocker on your end. Closing, but happy to re-open and investigate if it ends up being something different.
